system/autoware_diagnostic_graph_aggregator/doc/format/unit/diag.md

```diff
@@ -4,7 +4,9 @@ The `diag` object is a unit that refers to a specific status within the source d
 
 ## Format
 
-| Name   | Type     | Required | Description                            |
-| ------ | -------- | -------- | -------------------------------------- |
-| `type` | `string` | yes      | Specify `diag` when using this object. |
-| `diag` | `string` | yes      | The name of the diagnostic status.     |
+| Name      | Type     | Required | Description                            |
+| --------- | -------- | -------- | -------------------------------------- |
+| `type`    | `string` | yes      | Specify `diag` when using this object. |
+| `node`    | `string` | yes      | The name of the diagnostic status.     |
```

Comment:
Sorry if this is my misunderstanding, but perhaps "The name of the node status." not "The name of the diagnostic status.".

Reply:
To be precise, it is the node name part of the diagnostic status name.

Reply:
@soblin @sasakisasaki 
It may be a good idea to explain that the diagnostic name is assumed to be the output of diagnostic_updater and will be in the format `<node>: <diag>`.

Reply:
Also, if the diagnostic name does not follow the above format, if "node" name is an empty string, "diag" name will be used as the full name.
```
    node: ""
    name: arbitrary-diagnostic-status-name
```

Reply:
Setting `node` to empty strings was a workaround for old nodes that is no longer used in Autoware. To avoid confusion, please ignore this method.